### ISSUE TITLE
[Bug]: 1.19.3 crash

### ISSUE THREAD

**Issue by CrazyGinger42** [MinecraftschurliMods/Ars-Magica-Legacy#378]
### Describe the bug

I made a mod pack on curse forge and when it booted, the error came up that this mod can't run on 1.19.3. I made sure I was on the most current version of ars and that I had all of the libraries necessary. 

### Steps to Reproduce

1. Make a mod pack in curseforge with MC version 1.19.3
2. start and play pack
3. should bring you to the miecraft mod error screen

### Screenshots

![ARS Bug 4](https://user-images.githubusercontent.com/112960350/223713225-2782d52e-9625-41ea-b141-9089a9f2ff57.png)
![ARS Bug 1](https://user-images.githubusercontent.com/112960350/223713226-69338c54-a98e-445c-b504-7188b498b9c4.png)
![ARS bug 2](https://user-images.githubusercontent.com/112960350/223713227-6ba2aa52-bc2b-40ea-9765-b169bfde5f5c.png)
![ARS bug 3](https://user-images.githubusercontent.com/112960350/223713228-e4975843-f74e-4ae2-a383-6ee5a84026d2.png)

### Version

1.19.2-1.2.4

### Forge Version

44.1.0

**Comment by IchHabeHunger54** [comment]
The mod is not yet out on 1.19.3. You are using the 1.19.2 version of AM:L, which is not compatible with 1.19.3.

The issue probably stems from us accidentally releasing a 1.19.2 version that was incorrectly flagged as 1.19.3. This has been resolved already, but apparently CurseForge decided to ignore this.

TL;DR: No 1.19.3 yet, CF is bugged, remove the mod for now.

**Comment by Minecraftschurli** [comment]
just checked there is no version flagged with 1.19.3 on curseforge
